docs/ops/normalization/GroupNormalization_12.md

```diff
@@ -0,0 +1,79 @@
+# GroupNormalization {#openvino_docs_ops_normalization_GroupNormalization_12}
+
+@sphinxdirective
+
+**Versioned name**: *GroupNormalization-12*
+
+**Category**: *Normalization*
+
+**Short description**: Performs normalization of the input tensor according to the method described in https://arxiv.org/abs/1803.08494
+
+**Detailed description**
+
+The GroupNormalization operation performs the following transformation of the input tensor:
+
+  `y = scale * (x - mean) / sqrt(variance + epsilon) + bias`
```

Comment:
Consider usage of `.. math::` for this formula
Reference: https://github.com/openvinotoolkit/openvino/pull/17158